syntax = "proto3";
package PingPong;

message Ping {
    string message = 1;
    float delaySeconds = 2;
}

message Pong {
    string message = 1;
}

service PingPong {
    rpc SendPing(Ping) returns (Pong) {}
}
